content/articles/transferring-domain-away.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: Transferring a domain away from DNSimple
+excerpt: How to request a transfer code and transfer a domain from DNSimple to a different registrar.
+categories:
+- Domain Transfers
+---
+
+# Transferring a domain away from DNSimple
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## Before you start
+
+To transfer a domain to another registrar the standard procedure requires you to:
+
+1. unlock the domain at DNSimple
+1. request a transfer code from DNSimple
+1. start the transfer at the gaining registrar
+1. approve the transfer request sent via email
+
+The first two steps are responsibility of DNSimple. The other steps are responsibility of the gaining registrar, and we are not able to provide support for them.
```

Comment:
I'm thinking of something to use in place of "are responsibility of". I think "must happen at", but wanted to float it by others first.